### ISSUE TITLE
Offset Mod not working

### ISSUE THREAD

**Issue by Lightmaxifrvr** [p2r3/spplice#20]
When i try to play the offset mod the game opens and enables steam overlay (presumably becuase my portal 2 steam overlay is borked and dosent display anything) waiting for it to load it fills all of the circles and turns orange and after a while it crashes.

here is my OS taken from Steam.

Operating System Version:
"Arch Linux" (64 bit)
Kernel Name: Linux
Kernel Version: 6.5.9-zen2-1-zen
X Server Vendor: The X.Org Foundation
X Server Release: 12101009
X Window Manager: KWin
Steam Runtime Version: steam-runtime_0.20231024.64411

**Comment by p2r3** [comment]
game crashes really aren't easy to debug with this amount of information. have you tried just verifying the integrity of game files?

**Comment by Lightmaxifrvr** [comment]
Yep,everything validated successfully but the game still crashes.

**Comment by Lightmaxifrvr** [comment]
Weird

I alt tabbed out of the game 2 times and then it loaded?
